bin/start_fe.sh

```diff
@@ -132,13 +132,21 @@ fi
 
 # check java version and choose correct JAVA_OPTS
 java_version=$(jdk_version)
-final_java_opt=$JAVA_OPTS
+final_java_opt=$JAVA_OPTS_4G
+MemTotal=`free -g | grep "Mem"  | awk '{print $7}'`
```

Comment:
free is not available on all  Linux Distribution. we'd better to do `which free` before use it

Reply:
thx, updated